PSKoans/Koans/Constructs and Patterns/AboutRegularExpressions.Koans.ps1

```diff
@@ -0,0 +1,589 @@
+    <#
+        In regex, there are far more symbols with unique meanings than just quantifiers. What comes
+        next is not a definitive guide to EVERY SINGLE ONE, but rather an introduction to some of
+        the most common and useful special symbols in regex.
+        
+        This is ^\where things st\Art getting \weir\D$
+    #>
+    
+
+    Context '. (period)' {
+
+        It 'matches any character' {
+            # The . (period) matches literally any character
+    
+            $____ | Should -Be ('Lazy Sunday mornings' -match 'S.nday')
+        }
+
+        It 'starts looking kind of weird now' {    
+            '____' -match 'invi.a.ion' | Should -BeTrue
+        }
+    }
+
+    Context '\n' {
+
+        It 'matches new lines' {
+            <#
+                The \n symbol is the first regex symbol you've encountered that isn't just a single
+                character. Normally if you saw a lowercase "n" in a regex, it would just mean
+                "literally the letter n". When it's preceded by a backslash, however, that "n" takes
+                on a special meaning. In this case, \n matches new lines. Actually, it matches the
+                encoded character that indicates a new line.
+    
+                There are many more regex symbols that are single letters preceded by a backslash.
+                In fact, the backslash is probably the single most important character in all of
+                regex.
+            #>
+    
+            $multiLine = @"
+They might look
+similar and look
+soft, but you should
+never confuse a
+wild cougar for
+a domestic cat
+"@
+            $catPattern = '____\n'
+            $multiLine -match $catPattern | Should -BeTrue
+        }
+    }
+
+    Context '\d and \D' {
+        <#
+            The \d symbol matches digits (the numbers 0 - 9 inclusive), while the symbol \D
+            matches anything OTHER THAN a digit. Yes, regular expressions are case sensitive.
+            This "lower case for affirmative, upper case for negative" pattern is common among
+            regex symbols as you'll discover as you carry on.
+        #>
+
+        It 'matches digits' {
+            '__' -match '\d' | Should -BeTrue
+        }
+
+        It 'can require more than one occurence in a row' {
+            '__' -match '\d\d\d' | Should -BeTrue
+        }
+
+        It 'matches things that are NOT digits' {
+            '__' -match '\D' | Should -BeTrue
+        }
+    }
+
+    Context '\w and \W' {
+
+        It 'matches (non-)word characters' {
+            <#
+                \w matches word characters which are letters, numbers, and underscores. The \W
+                symbol matches anything else.
+
+                For these challenges, make sure you're using either \w or \W in your regular
+                expression. Otherwise, you're just cheating yourself!
+            #>
+            '***' -match '__' | Should -BeTrue
+        }
+
+        It 'sees no funky characters in here'
```

Comment:
```suggestion
        It 'sees only word characters in here' {
```
Ya broke it!

Also, this It name is vastly different to the rest of the file and feels a bit jarring. Feel free to reword it differently, but I don't think the current name fits well. 🙂